tools/karma_template.conf.js

```diff
@@ -68,6 +67,23 @@ const CUSTOM_LAUNCHERS = {
     os: 'Windows',
     os_version: '10'
   },
+  bs_chrome_mac_webgpu: {
+    base: 'BrowserStack',
+    browser: 'chrome',
+    browser_version: 'latest',
+    os: 'OS X',
+    os_version: 'High Sierra',
+    flags: [
+      '--enable-features=Vulkan,UseSkiaRenderer',
+      '--use-vulkan=native',
+      '--enable-unsafe-webgpu',
+      '--disable-vulkan-fallback-to-gl-for-testing',
+      '--disable-vulkan-surface',
+      '--disable-features=VaapiVideoDecoder',
+      '--ignore-gpu-blocklist',
+      '--use-angle=vulkan',
```

Comment:
I probably don't need all these flags

Reply:
My impression was BrowserStack couldn't support browser flags so we couldn't support WebGPU. Is there any new improvement recently? 
Maybe we only need "--enable-unsafe-webgpu" on macOS as WebGPU should use Metal instead of Vulkan.
If BrowserStack can support Chrome Canary, we don't need any flags at all.

Reply:
This wasn't possible before, but [BrowserStack recently added support for Chrome flags](https://www.browserstack.com/docs/automate/javascript-testing/chrome-flags), so we're able to change this setting in the Karma launcher just like you'd do it when running locally. In addition to letting us test WebGPU now I expect this will also be very useful for testing new web APIs like WebNN in the future.

Reply:
> I probably don't need all these flags

I think only `'--enable-unsafe-webgpu',` is needed for Mac. It seems that other flags are for Linux. And properly many of them can be removed now for Linux.

Reply:
It's really great to hear about this, and thank you for raising this feature request to BrowserStack!